.rubocop.yml

```diff
@@ -30,6 +25,9 @@ Style/EmptyElse:
 Lint/AmbiguousRegexpLiteral:
   Enabled: false
 
+Lint/AmbiguousBlockAssociation:
+  Enabled: false
```

Comment:
Why it is disabled globally? For what cases it fail?

Reply:
In case when lambda or proc is specified. it does not fail if add it in `set :deploy_to, (proc { "some_url}" })` . The code looks not very nice after that.